v1/python-sdk/tutorials/automl-with-azureml/automl-nlp-multiclass/automl-nlp-text-classification-multiclass.ipynb

```diff
@@ -104,7 +104,7 @@
    },
    "outputs": [],
    "source": [
-    "print(\"This notebook was created using version 1.39.0 of the Azure ML SDK\")\n",
+    "print(\"This notebook was created using version 1.46.0 of the Azure ML SDK\")\n",
```

Comment:
Should have shared this sooner, but last week when Miseon and I were discussing, we realized that it's better to remove this line else we'd need to update regularly.